engwales-plead.sty

```diff
@@ -25,3 +26,10 @@
   \begin{flushright}\uline{Defendant}\end{flushright}
   \endgroup
 }
+
+%% Allow for you to list items with the number in the margin
+\newcounter{point}
+\newenvironment{point}[1][]
+{\begin{enumerate}[leftmargin=4em,labelsep=1.5em]\refstepcounter{point}
+   \item[\thepoint.] #1
```

Comment:
move the label format to list

Reply:
I'm not quite sure what you mean, but gave it a shot. You can now pass enumerate options through the `\begin{point}[stuff]`.

Reply:
sorry for the confusion. i meant defining the new list with `\newlist` and `\setlist` provided by enumitem, and use `label=\arabic*.` to set the label format.